persistent/test/main.hs

```diff
@@ -212,7 +214,7 @@ main = hspec $ do
         it "mid-token quote in later token" $
             parseLine "foo bar baz=(bin\")" `shouldBe`
                 Just
-                    ( Line 0
+                    ( Line 0 $ nonEmptyOrFail
```

Comment:
For the test suite, we could enable `OverloadedLists`, and then we can use list literal syntax for `NonEmpty a` - it's a gross kludge to write `[] :: NonEmpty Int`, but, lmao, it does make the test syntax nicer.